google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClientTest.java

```diff
@@ -585,32 +578,13 @@ public void testUpdateBackup() {
   @Test
   public void testRestoreTable() throws ExecutionException, InterruptedException {
     // Setup
+    Mockito.when(mockStub.restoreTableOperationCallable())
+        .thenReturn(mockRestoreTableOperationCallable);
+
     Timestamp startTime = Timestamp.newBuilder().setSeconds(1234).build();
     Timestamp endTime = Timestamp.newBuilder().setSeconds(5678).build();
     String operationName = "my-operation";
     RestoreTableRequest req = RestoreTableRequest.of(CLUSTER_ID, BACKUP_ID).setTableId(TABLE_ID);
-    Mockito.when(mockRestoreTableCallable.futureCall(req.toProto(PROJECT_ID, INSTANCE_ID)))
```

Comment:
a bit confused why this can be removed?

Reply:
It seems the only mock actually used in this test was `mockRestoreTableOperationCallable`. This was getting flagged as unused in a warning.

Reply:
oh it seems like its a duplicate of mockOperationResult below